### DISCUSSION TITLE
Motor support stops in between and comes back with a delay what is that.

### DISCUSSION THREAD

**Discussion by Larswagner1987** [danielnilsson9/bbs-fw#118] [Q&A] [answered]
Hello

I have a problem I have the version on my 2 e-bikes and it always happens that the motor no longer supports it and then it happens again more often, unfortunately there is no error message or the same. Engine is a BBS02B.

thanks in advance

**Comment by silenec** [comment, marked as answer]
Do you mean like this? https://github.com/danielnilsson9/bbs-fw/issues/114

**Reply by silenec** [reply]
Yeah that does sound like the same issue actually.

**Reply by Larswagner1987** [reply]
Ja genau , sorry habe ich nicht gesehen das da jemand schon was geschrieben hat .

@danielnilsson9 ich habe noch ein bbs02 Controller, willst du den haben das du es testen kannst oder die Daten auslesen kannst ??? Würde dir den zusenden .

**Reply by Larswagner1987** [reply]
@danielnilsson9 The problem now is that I can no longer get the original version on it and now I have yours on it, at the beginning everything went well, including the test drive without errors, I will play your version again right away and then test it again and if so I have this error, you will immediately connect my laptop and see if there is an error. or do you have another tip what I can do or what could be the problem?

**Reply by danielnilsson9** [reply]
There is no way of reading that error log after it occurs, you have to have the computer connected while it's happening. Just lower you max current and check if it goes away.

**Reply by Larswagner1987** [reply]
@danielnilsson9 ich habe schon den Strom von 25a auf 22a ich werde mal 20 a versuchen und melde mich wieder

**Reply by danielnilsson9** [reply]
Ok, that weird, then I don't know

**Reply by Larswagner1987** [reply]
@danielnilsson9  look here maybe this can help you

https://github.com/philippsandhaus/bafang-python/blob/master/README.md
